js/ui/overlay/ui.overlay.js

```diff
@@ -479,6 +485,22 @@ const Overlay = Widget.inherit({
         this._refresh();
     },
 
+    _renderWrapperAttributes(wrapperAttr = this.option('wrapperAttr') || {}) {
+        const attributes = extend({}, wrapperAttr);
+        const classNames = attributes.class;
+        delete attributes.class;
+        if(this._oldAttributes) {
```

Comment:
Let's get rid of `_oldAttributes` and pass `previousValue` from option change handler

Reply:
done